Exception Handling/ManipulacaoCSV/com.manipulacao-csv.java/ListaPessoas.java

```diff
@@ -0,0 +1,62 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class ListaPessoas {
+
+    public static List<Pessoa> PreencheLista(String arquivoCSV) {
```

Comment:
o que acha de darmos um nome mais autodescritivo para esse método?? 
```suggestion
    public static List<Pessoa> lerCsvEConverteParaPessoas(String arquivoCSV) {
```

pensei em algo do tipo XD

pensei em algo nesse sentido XD